fireplace-swing/src/main/java/io/github/bric3/fireplace/flamegraph/FlameGraphPainter.java

```diff
@@ -60,11 +66,14 @@
     private final int internalPadding = 2;
     protected boolean paintDetails = true;
 
-
     public FlameGraphPainter(List<FrameBox<T>> frames,
                              List<Function<T, String>> nodeToTextCandidates,
                              Function<T, String> rootFrameToText,
                              Function<T, Color> frameColorFunction) {
+
+        this.frameLabelFont = new Font(Font.SANS_SERIF, Font.PLAIN, 12);
+        this.frameLabelFontItalic = new Font(Font.SANS_SERIF, Font.ITALIC, 12);
```

Comment:
I'd rather name about the intent of this font, not sure about the name something like : `largerFrameLabelFont`

Reply:
I changed the name to `frameLabelFontForPartialFrames`